### ISSUE TITLE
Wrong variable type - 2.2.1 - Variables and Mutability

### ISSUE THREAD

**Issue by TAdev0** [cairo-book/cairo-book.github.io#335]
Hi :) 

I  think i noticed a little error when reading the common programming concepts part. 

At some point, last part of 2.2.1 says "  For example, say our program performs a type conversion between the u64 and felt252 types. " . Then, the first example is :

```
use debug::PrintTrait;
use traits::Into;
fn main() {
    let x = 2;
    x.print();
    let x: felt252 = x.into(); // converts x to a felt, type annotation is required.
    x.print()
}

The first x variable has a u64 type while the second x variable has a felt252 type

```

shouldn't the first line of the function be : ```let x : u64 = 2;  ```?

After that, there's another example :

```
use debug::PrintTrait;
use traits::Into;
fn main() {
    let mut x: u32 = 2;
    x.print();
    x = 100_felt252;
    x.print()
    
}

The error says we were expecting a u64 (the original type) but we got a different type:

```

Actually the code  first defines x as a u32, and the screen of CLI while trying scarb cairo-run indeed prints an unexpected argument type error, with u32 expected.

I guess all u64 in this section should be changed for u32 , so that the CLI screen will remain correct with the overall context!

**Comment by enitrat** [comment]
> shouldn't the first line of the function be : let x : u64 = 2;  ?

yes!

> I guess all u64 in this section should be changed for u32 , so that the CLI screen will remain correct with the overall context!

or the opposite, feel free to submit a PR !

**Comment by enitrat** [comment]
Solved by #336
